tests/testing.py

```diff
@@ -54,7 +54,7 @@ def _scrub_sensitive_response_info(response):
 
 my_vcr = vcr.VCR(
     cassette_library_dir=recording_path,
-    record_mode="once",
+    record_mode=settings.RECORD_MODE,
```

Comment:
Can we have some documentation (maybe in a file, not in the actual documentation) of how to add new tests with recorded server responses?